projects/ngx-sub-form/src/lib/shared/ngx-sub-form-utils.ts

```diff
@@ -25,14 +26,18 @@ export type ControlsType<T> = {
 
 export type OneOfControlsTypes<T = any> = ControlsType<T>[keyof ControlsType<T>];
 
-// @deprecated
+/**
+ * @deprecated
+ */
 export type FormErrorsType<T> = {
   [K in keyof T]-?: T[K] extends any[] ? (null | ValidationErrors)[] : ValidationErrors;
 };
 
 export type FormUpdate<FormInterface> = { [FormControlInterface in keyof FormInterface]?: true };
```

Comment:
You can also mark that one as deprecated BTW.

It's used only by `OnFormUpdate` which itself [has been deprecated for a while](https://github.com/cloudnc/ngx-sub-form/blob/master/projects/ngx-sub-form/src/lib/ngx-sub-form.types.ts#L5-L9)

Reply:
ah cool, will follow up

Reply:
did here as there was a conflict